scripts/run/shutdown.sh

```diff
@@ -45,17 +45,20 @@ function load_env_variables() {
 # Globals:
 #   None
 # Arguments:
-#   timestamp - A timestamp in the form "hh:mm:ss" to be checked against the current time.
+#   $1 - A timestamp in the form "hh:mm:ss" to be checked against the current time.
```

Comment:
Is there a reason you changed this to $1? I feel like this could be named better

Reply:
I did this for consistency with how @jbesoto did it since the argument is actually the first input which is what `$1` represents.